dex/src/main/scala/com/wavesplatform/dex/api/http/routes/MatcherApiRoute.scala

```diff
@@ -1174,6 +1178,31 @@ class MatcherApiRoute(
     }
   }
 
+  @Path("/debug/address/{address}")
+  @ApiOperation(
+    value = "Get state (balances, placement queue by address",
+    httpMethod = "GET",
+    authorizations = Array(new Authorization(SwaggerDocService.apiKeyDefinitionName)),
+    tags = Array("debug"),
+    response = classOf[HttpAddressActorState]
+  )
+  @ApiImplicitParams(
+    Array(
+      new ApiImplicitParam(name = "address", value = "Address", dataType = "string", paramType = "path")
+    )
+  )
+  def getAddressActorState: Route = (path("address" / AddressPM) & get & withAuth) {
+    addressOrError =>
+      withAddress(addressOrError) { address =>
+        complete {
+          askMapAddressActor[GetState](address, GetCurrentState) {
+            reply =>
+              Map("tradable" -> reply.tradable.toJson, "reserved" -> reply.reserved.toJson, "placementQueue" -> reply.placementQueue.toJson)
```

Comment:
1. Again, let's create an entity class like `HttpV0OrderBook`. The motivation part is described in another PR: https://github.com/wavesplatform/matcher/pull/482#pullrequestreview-617290023
2. Also we need the full balance information (see `AddressBalance` class), because we already know `tradable` and `reserved` balances by other API methods. You need to create a copy of `AddressBalance`: `HttpAddressBalance` with simpler fields: `Map` instead of `NonNegativeMap`/`PositiveMap`/`NonPositiveMap`/`NegativeMap`, `String` instead of `ExchangeTransaction.Id`

Reply:
fiexd